core/src/main/java/org/opensearch/sql/expression/ExpressionNodeVisitor.java

```diff
@@ -15,8 +14,9 @@
 
 /**
  * Abstract visitor for expression tree nodes.
- * @param <T>   type of return value to accumulate when visiting.
- * @param <C>   type of context.
+ *
+ * @param <T> type of return value to accumulate when visiting.
+ * @param <C> type of context.
```

Comment:
technically the spacing on this one is still fine because both lines have the same length but still should have pre

Reply:
I don't think it makes a difference?